aconcli/service/aconclient_http.go

```diff
@@ -226,6 +253,130 @@ func multipartManifestField(w *multipart.Writer, fieldname, filename string) err
 	return err
 }
 
+func (c *AconClientHttp) setRequestAuthHeader(req *http.Request) error {
+	if c.noAuth {
+		return nil
+	}
+	if err := c.fetchSessionKey(); err != nil {
+		return fmt.Errorf("failed to get auth token: %v", err)
+	}
+	req.Header.Add("Authorization", c.sessionkey)
```

Comment:
Just to confirm my understanding, that `c.sessionkey` includes the authorization type - `Basic` in this case?